<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="headImgToolBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainLayout"/>
  </widget>
  <widget class="QTextEdit" name="signNameTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:1px;
font: 9pt &quot;Courier New&quot;;
border-color: rgb(105, 255, 143);
border-style:solid; </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="searchTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:1px;
font: 9pt &quot;Courier New&quot;;
border-color: rgb(93, 255, 131);
border-style:solid; </string>
   </property>
  </widget>
  <widget class="QToolButton" name="searchToolBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="mainTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>281</width>
     <height>391</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>联系人</string>
    </attribute>
    <widget class="QTreeWidget" name="friendTreeWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>291</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeView {
    show-decoration-selected: 1;
}

QTreeView::item {
     border: 1px solid #d9d9d9;
    border-top-color: transparent;
    border-bottom-color: transparent;
}

QTreeView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
}

QTreeView::item:selected {
    border: 1px solid #567dbc;
}

QTreeView::item:selected:active{
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
}

QTreeView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
}
</string>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="sysSettingToolBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sysHelpToolBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
